aiida/engine/processes/builder.py

```diff
@@ -23,11 +23,13 @@ class ProcessBuilderNamespace(collections.abc.MutableMapping):
     """
 
     def __init__(self, port_namespace):
-        """Dynamically construct the get and set properties for the ports of the given port namespace.
+        """Dynamically construct the get and set properties for the ports of the given port
+        namespace.
 
-        For each port in the given port namespace a get and set property will be constructed dynamically
-        and added to the ProcessBuilderNamespace. The docstring for these properties will be defined
-        by calling str() on the Port, which should return the description of the Port.
+        For each port in the given port namespace a get and set property will be constructed
+        dynamically and added to the ProcessBuilderNamespace. The docstring for these
+        properties will be defined by calling str() on the Port, which should return the
+        description of the Port.
```

Comment:
Did a linter adjust the length of all docstrings or did you do this manually? Anyway, please revert. We are maintaining a 120 character limit for the entire project, including docstrings. Doesn't make sense to start changing this here.

Reply:
Ok, maybe something in my pylint got messed up, but it didn't let me commit with lines longer than a 100 chars (that is why I originally put the `# pylint: disable=line-too-long` that I was requested to remove).

Reply:
If there is a problem with your dev installation, it would be more sensible to try and fix that, instead of adapting the code to the broken environment, wouldn't it? Anyway, you should simply update your installation by running `pip install -e .[pre-commit]` which will install the correct version of `pylint`. The configuration has moved to `pyproject.toml` instead of the `.pylintrc` file, but this is only supported starting from `pylint==2.6`,